airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/debezium/internals/DebeziumRecordIterator.kt

```diff
@@ -193,15 +207,22 @@ class DebeziumRecordIterator<T>(
         LOGGER.info {
             "Time since last hb_pos change ${timeElapsedSinceLastHeartbeatTs.toSeconds()}s"
         }
+
         // wait time for no change in heartbeat position is half of initial waitTime
+        if (isTest()) {
+            return false
```

Comment:
should this return `true` as we need this function to trigger in test?

Actually could the whole function be replaced with the following?

```
private fun heartbeatPosNotChanging(): Boolean {
   return isTest();
}
```

Reply:
and we'd better add some comments here also to clarify the intention of this function

Reply:
I did want to keep the original function for logging - as we roll this out, it could be useful to see how many syncs in production are being stalled due to hb issues (even though they won't be killed during the sync itself)

Reply:
Actually, I will clean this up. I'm not sure that the log is that useful